script.js

```diff
@@ -0,0 +1,59 @@
+let books = [];
+let i = 0;
+
+const form = document.querySelector('.addBooks');
+const bookList = document.getElementById('booksList');
+const submit = document.getElementById('addBooks');
+
+function listBooks(books) {
+  return `
+    <li>${books.title}</li>
+    <li>${books.author}</li>
+    <button type='button' id='${books.id}' class='remove-btn'>Remove</button>`;
+}
```

Comment:
- [ ] Kindly change this regular function to an es6 arrow function. As stated in the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.).